CMakeLists.txt

```diff
@@ -27,9 +27,6 @@ endif()
 
 get_filename_component(AWS_NATIVE_SDK_ROOT "${CMAKE_CURRENT_SOURCE_DIR}" ABSOLUTE)
 
-# git is required for Android builds and building third-party dependencies
```

Comment:
ok, this is all in our cmake shim module

Reply:
they used git during the execution of cmake to fetch some dependencies (aws-c-common, aws-checksum and aws-c-event-stream). but i changed them to regular submodules and removed the git dependency